### PR TITLE
Update bioRxiv link/title to Nature Methods

### PR DESCRIPTION
**Pull request by joshmoore**
See dx.doi.org/10.1038/nmeth.4326

### CODE REVIEW

**Comment by joshmoore** [comment]
https://idr.openmicroscopy.org/about/submission.html updated manually.

File: submission.html [hunk 1]
```diff
@@ -294,9 +294,9 @@ <h1 class="hero-main-header">Submission of datasets to the IDR</h1>
 
                         <p style="display:inline-block;margin-left:5em;">
                             <i>
-                                Williams et al (2017) The Image Data Resource: A Scalable Platform for Biological Image Data Access,
-                                Integration, and Dissemination <a href="http://biorxiv.org/content/early/2016/11/24/089359">
-                                http://biorxiv.org/content/early/2016/11/24/089359</a> .
+                                Williams et al (2017) The Image Data Resource: a bioimage data integration and publication platform,
```
Comment (manics) [inline submission.html]: `The` or no `The`?

Comment (joshmoore) [inline submission.html]: No `The` in new title.

**Comment by joshmoore** [comment]
Done (also live)

**Comment by manics** [comment]
Tested with IDR/deployment:
```diff
diff --git a/ansible/group_vars/proxy-hosts.yml b/ansible/group_vars/proxy-hosts.yml
index 5f4f2f6..418c5a6 100644
--- a/ansible/group_vars/proxy-hosts.yml
+++ b/ansible/group_vars/proxy-hosts.yml
@@ -244,6 +244,7 @@ nginx_proxy_streams: >

 jekyll_build_sourcedir: /home/omero/about
 jekyll_build_root: /srv/www/html
-jekyll_build_git_repo: https://github.com/IDR/idr.openmicroscopy.org.git
+jekyll_build_git_repo: https://github.com/joshmoore/idr.openmicroscopy.org.git
+jekyll_build_git_branch: nature
 jekyll_build_force_git: True
 jekyll_build_baseurl: /about
```
screen shot 2017-06-21 at 10 52 23

**Comment by joshmoore** [comment]
Cheers. I'll add a tag with today's date.

**Pull request merged**